src/Components/EventSchedule.jsx

```
@@ -0,0 +1,189 @@
+  const dayEvents = scheduleData.filter(event => event.day === selectedDay);
+  
+  // Find current event based on time
+  useEffect(() => {
+    const findCurrentEvent = () => {
+      const now = new Date();
+      // Logic to find current event
+      // ...
+      const currentEvent = dayEvents.find(event => {
+        
+        const eventTime = new Date(`${event.day}, 2025 ${event.time}`);
+        const [startHour, startMinute] = event.time.split(':').map(num => parseInt(num));
```

ESLint

[error] 'startHour' is assigned a value but never used.

(no-unused-vars)